### ISSUE TITLE
mongoDB + jhipster UML

### ISSUE THREAD

**Issue by mayurbais** [jhipster/jhipster-uml#257]
Hi,
I have my XMI generated out of modelio. I deleted all the composition /association relationships from my UML , so that the generated XMI would be acceptable when I use jhipster-uml generated on my jhipster project, which is using mongoDB. Which is now acceptable. 

### }

I have added attribute with name building and of type "Building" . Both Room and Building are classes in my UML diagram. There are many such instances. I replaced the  attribute name to buildingId and Type to String. It works!. 

can we not have any attribute of custom Type in uml for Jhipster-uml to successfully generate entities for mongoDB. something like embedded document? 

### Versions 

| JHipster           | JHipster UML           | Editor & version |
| ------------------ | ---------------------- | ---------------- |
| `4.6.0` |    `2.0.3`   |           Modelio-3.6       |

**Comment by MathieuAA** [comment]
Hi @mayurbais, can you please provide your XML file?
By the way, I know Modelio v3.6 is quite buggy, can you please update to at least v3.6.1 to see if the problem's solved?

**Comment by mayurbais** [comment]
Hello MathieuAA, I updated to 3.6.1 and it still has the error :
![image](https://user-images.githubusercontent.com/9245684/29455780-60cf62c0-83c7-11e7-9487-ff330c55b5e6.png)

**Comment by MathieuAA** [comment]
There's no special guideline, it's pretty straightforward as you just don't have to use relationships.
I'll try to see if there's anything I can do to make easier the process, but I doubt it.

**Comment by mayurbais** [comment]
yeah...I was using modeilo version 3.4 earlier and all was well in life..Then I updated to modelio 3.6.1. and it started to fall apart. I compared the xmi generated by both the versions. The xml:id which are assigned to various packaged elements were duplicated by 3.6.1.. and I have moved back to 3.4 and have to do the re-work on removing relationships in this. 
(FYI..xmis generated by these seperate versions are also not compatible with each other..so can no do the import of xmi too...)
Thanks for your time..looks like I will have toburn midnight oil for this..again...
